applications/ConstitutiveLawsApplication/custom_utilities/advanced_constitutive_law_utilities.h

```diff
@@ -478,6 +479,75 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) AdvancedConstitutiveLawUtilities
         BoundedMatrixVoigtType &rNewOperator
     );
 
+    /**
```

Comment:
Test for this method will be nice

Reply:
yes, at the end of the PR i'll add tests for everything